src/Microsoft.PowerApps.TestEngine/System/UserInputException.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerApps.TestEngine.System
+{
+    public class UserAppException : Exception
```

Comment:
Shouldn't this be in teh "UserAppException.cs" file?

Reply:
Addressing it, good catch.